index.html

```diff
@@ -990,32 +656,24 @@ <h2>
     Examples
   </h2>
   <pre class="example js" title="Adjusting the number of video feeds based on CPU usage">
-    const observer = new ComputePressureObserver(computePressureCallback);
+    let timerId = -1;
+    function pressureChange(entries) {
+      clearTimeout(timerId);
```

Comment:
Let's clear only if `timerId` has been set.

Reply:
I can do that, but clearTimeout with negative values doesn't do anything

Reply:
Done